packages/nextlove/src/generators/lib/parse-routes-in-package.ts

```diff
@@ -36,6 +36,10 @@ export const parseRoutesInPackage = async (opts: {
     filepaths.map(async (p) => {
       const { default: routeFn } = await require(path.resolve(p))
 
+      if (routeFn._routeSpec.metaData.ignoreEndpoint) {
+        return
+      }
+
       if (routeFn) {
```

Comment:
Considering the `if (routeFn)` check, I assume `routeFn` can be undefined. If so, then we should check the metadata inside the `if (routeFn)` check.
Another thing, is `metaData` always present? Empty object by default? If not, then we should use optional chaining operator. I suppose `ignoreEndpoint` can also be undefined. Considering the above we should access the property as `routeFn._routeSpec?.metaData?.ignoreEndpoint`.

```suggestion
      if (routeFn) {
        if (routeFn._routeSpec?.metaData?.ignoreEndpoint) {
          return
        }  
```